source/local/local.go

```diff
@@ -37,6 +38,13 @@ const Name = "local"
 // LabelFeature of this feature source
 const LabelFeature = "label"
 
+// ExpiryDateKey is the key of this feature source indicating
+// when features should be removed.
+const ExpiryDateKey = "# +expiry-time"
+
+// CommentsPrefix defines the prefix of comments that should be parsed
+const CommentsPrefix = "# +"
```

Comment:
nit: this is not comment prefix (comments start with `#`). Should be e.g. `TagPrefix` or `OptPrefix` or `DirectivePrefix`  or smth.

Reply:
No no 😅 What I tried to suggest is to not have the `# +` prefix included in the `ExpiryTimeKey` const. All tags/opts/directives would start with `# +` so we don't need to repeat it